src/components/TodoApp.jsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import uuid from 'uuid';
+import TodoForm from './TodoForm/TodoForm';
+import TodoList from './TodoList/TodoList';
+import TodoFilters from './TodoFilters/TodoFilters';
+
+class TodoApp extends React.Component {
+  state = {
+    todos: [],
+    activeFilter: 'all',
+  }
+
+  handleSubmitForm = (todo) => {
+    const idForTodo = uuid.v4();
+    const whiteSpaceRegExp = new RegExp(/^[^\s]/);
```

Comment:
он у нас здесь всего один, можно просто
```suggestion
    const pattern = new RegExp(/^[^\s]/);
```